openapi2conv/openapi2_conv.go

```diff
@@ -121,6 +121,51 @@ func ToV3PathItem(swagger *openapi2.Swagger, pathItem *openapi2.PathItem) (*open
 	return result, nil
 }
 
+func ToV3RequestBodyFormData(parameters []*openapi2.Parameter) *openapi3.RequestBodyRef {
+	if len(parameters) == 0 || parameters[0].In != "formData" {
```

Comment:
Just a quick question, this logic implies that any `formData` parameters will only be converted if the very first parameter is a `formData`. This means that if the `formData` parameter is second in the list it'll not be converted?

Also, the logic below implies that if there was any `formData` parameters that any other parameters will be ignored? (https://github.com/getkin/kin-openapi/pull/233/files#diff-29b4895fc45bb3c5db76a7a9635a745eR186-R200)

Reply:
Indeed it looks like the spec does not enforce this logic. We need a PR that processes the other parameters as well.

Reply:
I am working on that. I totally omited the fact that other parameters are possible too, since https://swagger.io/docs/specification/2-0/file-upload/ examples not use any and my use-case didn't need any of them.

Reply:
Addressed in https://github.com/getkin/kin-openapi/pull/239